File: 프로그래머스/Lv.0/181925. 수 조작하기 2/수 조작하기 2.java
class Solution {
    public String solution(int[] numLog) {
        String answer = "";
        
        for(int i = 1; i < numLog.length; i++){
            int a = numLog[i-1];
            int b = numLog[i];
            int cal = b - a;
            
            if(cal == 1){
                answer += "w";
            }else if(cal == -1){
                answer += "s";
            }else if(cal == 10){
                answer += "d";
            }else if(cal == -10){
                answer += "a";
            }
            
        }
        
        return answer;
    }
}

File: 프로그래머스/Lv.0/181925. 수 조작하기 2/수 조작하기 2.py
def solution(numLog):
    answer = ''
    key = { 1 : "w", -1 :"s", 10 :"d", -10:"a"}
    for i in range(1, len(numLog)):
        answer += key[numLog[i] - numLog[i-1]]
        
    return answer